src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.FSharp.xaml

```diff
@@ -33,6 +33,26 @@
                 HelpUrl="https://docs.microsoft.com/dotnet/fsharp/language-reference/compiler-options"
                 Category="General" />
 
+  <EnumProperty Name="FSharpPreferNetFrameworkTools"
+              DisplayName="Framework tools"
+              Description="Compiler to use when building the project"
+              Category="General">
+    <EnumValue Name="true"
+             DisplayName=".NET Framework" />
+    <EnumValue Name="false"
+             DisplayName=".NET SDK" />
+  </EnumProperty>
+
+  <EnumProperty Name="FSharpPrefer64BitTools"
+            DisplayName="Prefer 64 bit tools"
+            Description="Use a 64 bit or 32 bit version of the compiler when available"
+            Category="General">
+    <EnumValue Name="false"
+           DisplayName="32 bit compiler" />
+    <EnumValue Name="true"
+           DisplayName="64 bit compiler" />
```

Comment:
This label seems potentially confusing. There is no guarantee you'll end up with a 64-bit compiler. I think a check box would be clearer.

Reply:
I will probably make it a checkbox, however it may be a bit confusing at the moment, since we don't (yet) set the default value to 'true', but it is rather empty, see:

https://github.com/dotnet/fsharp/blob/release/dev17.0/src/fsharp/FSharp.Build/Microsoft.FSharp.Targets#L64-L69